RELEASING.md

```diff
@@ -0,0 +1,190 @@
+after branching v4.0, `composer.json` in the master branch may still read:
+
+```
+"branch-alias": {
+    "dev-master": "4.0.x-dev"
+}
+```
+
+The above would be changed to:
+
+```
+"branch-alias": {
+    "dev-master": "4.1.x-dev"
+}
+```
+
+Commit this change:
+
+```console
+$ git commit -m "Master is now 4.1-dev" composer.json
+```
+
+### After releasing a new minor version
+
+After a new minor version is released (i.e. `master` was tagged), a maintenance
+branch should be created for future patch releases:
+
+```console
+$ git checkout -b vX.Y
+$ git push mongodb vX.Y
+```
+
+Update the master branch alias in `composer.json`:
+
+```diff
+ "extra": {
+   "branch-alias": {
+-    "dev-master": "4.0.x-dev"
++    "dev-master": "4.1.x-dev"
+   }
+ },
+```
+
+Commit and push this change:
+
+```console
+$ git commit -m "Master is now X.Y-dev" composer.json
+$ git push mongodb
+```
+
+### After releasing a patch version
+
+If this was a patch release, the maintenance branch must be merged up to master:
+
+```console
+$ git checkout master
+$ git pull mongodb master
+$ git merge vX.Y --strategy=ours
+$ git push mongodb
+```
+
+The `--strategy=ours` option ensures that all changes from the merged commits
+will be ignored. This is OK because we previously ensured that the `master`
+branch was up-to-date with all code changes in this maintenance branch before
+tagging.
+
+
+## Publish release notes
+
+The following template should be used for creating GitHub release notes via
+[this form](https://github.com/mongodb/laravel-mongodb/releases/new).
+
+```markdown
+The PHP team is happy to announce that version X.Y.Z of the MongoDB integration for Laravel is now available.
+
+**Release Highlights**
+
+<one or more paragraphs describing important changes in this release>
+
+A complete list of resolved issues in this release may be found in [JIRA]($JIRA_URL).
+
+**Documentation**
+
+Documentation for this library may be found in the [Readme](https://github.com/mongodb/laravel-mongodb/blob/$VERSION/README.md).
+
+**Installation**
+
+This library may be installed or upgraded with:
+
+    composer require mongodb/laravel-mongodb:X.Y.Z
+
+Installation instructions for the `mongodb` extension may be found in the [PHP.net documentation](https://php.net/manual/en/mongodb.installation.php).
+```
+
+The URL for the list of resolved JIRA issues will need to be updated with each
+release. You may obtain the list from
+[this form](https://jira.mongodb.org/secure/ReleaseNote.jspa?projectId=22488).
+
+If commits from community contributors were included in this release, append the
+following section:
+
+```markdown
+**Thanks**
+
+Thanks for our community contributors for this release:
+
+ * [$CONTRIBUTOR_NAME](https://github.com/$GITHUB_USERNAME)
+```
+
+Release announcements should also be posted in the [MongoDB Product & Driver Announcements: Driver Releases](https://mongodb.com/community/forums/tags/c/announcements/driver-releases/110/php) forum and shared on Twitter.
```

Comment:
Do you want to remove this? Or is it consistent with how other languages announce their extra-driver releases (e.g. Ruby announcing Mongoid)?

Reply:
Ruby does indeed post these for Mongoid as well. IIRC the announcements should cover all products maintained by the DBX team, hence the inclusion here.